src/OpenConext/ProfileBundle/Resources/views/MyServices/AttributeList/idp.html.twig

```diff
@@ -1,10 +1,13 @@
 <p>{{ 'profile.my_services.attribute_release_description'|trans }}</p>
-
+{% set showMotivations = specifiedConsent.hasMotivations %}
 <table class="mdl-data-table">
     <thead>
     <tr>
         <th>{{ 'profile.table.attribute_name'|trans }}</th>
         <th>{{ 'profile.table.attribute_value'|trans }}</th>
+        {% if showMotivations %}
+            <th>{{ 'profile.table.motivation'|trans }}</th>
```

Comment:
Is 'motivation' what we want to call it to end users? Maybe take a screenshot and hang it in pivotal for surf to bikeshed.